specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/examples/GetAGalleryImageVersionWithReplicationStatus.json

```diff
@@ -0,0 +1,109 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2019-12-01",
+    "galleryName": "myGalleryName",
+    "galleryImageName": "myGalleryImageName",
+    "galleryImageVersionName": "1.0.0",
+    "galleryImageVersion": {
```

Comment:
i know it may appear in previous examples. but it looks to me this "galleryImageVersion" field is not needed.

Reply:
do you want to remove it in all the examples? what would you replace it with?

Reply:
yeah, remove them from all the examples